Feat/adding subgraph hook with axios

RafaDSan opened the pull request:
closes #74  

- [x] Added script utilizing `axios `and `GraphQL `(subgaph) to query data from the Dapp (It's working as shown in IMG1)
- [x] `axios `and `dotenv `dependencies added to the `package.json`
- [x] `.env-example` file updated to have subgraph infos



**IMG 1**
![image](https://github.com/blockful-io/swaplace-dapp/assets/102108966/b26a5384-519b-41e1-b989-315e58e57f71)

Resulting of the bash command `node lib/client/hooks/subgraphQueries.ts`

**OBS** Corrected PR to point to the `develop` branch!

RafaDSan commented:
@heronlancellot new PR done!

heronlancellot commented on .env-example:
Hey man! We already have the .env.example file in the project, can you remove the modifications in the .env-example and update the .env.example? 🙏
Thanks for updating that!

heronlancellot reviewed (changes requested):
- [ ] update the .env.example file & remove .env-example
- [ ] update package.json with the latest modifications

RafaDSan commented:
> Hey @RafaDSan! Thanks for updating the PR! I added one review in .env-example
> 
> BTW, I can't merge yet because have some conflicts in package.json, Can you please update the package.json as well? we updated it like 5' before your PR 😂 Sorry for that. ^^
> 
> * [x]  update the .env.example file & remove .env-example
> * [x]  update package.json with the latest modifications

Package.json and .env.example updated! @0xneves or @heronlancellot could check it, please?

0xneves reviewed (commented):
The issue here @RafaDSan is that you synced your branch with the '[main](https://github.com/blockful-io/swaplace-dapp/tree/main)' branch, but the branch to be sync is the '[develop](https://github.com/blockful-io/swaplace-dapp/tree/develop)'. Notice the difference in the package.json and the CODEOWNERS file as well
